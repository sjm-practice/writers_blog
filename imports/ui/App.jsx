/* eslint-disable arrow-parens, object-curly-newline */
import React, { Component } from "react";
import PropTypes from "prop-types";
import { Typography, Paper } from "@material-ui/core";
import { withStyles } from "@material-ui/core/styles";
import styles from "./styles";

// App component - represents the whole app
export default withStyles(styles)(
  class App extends Component {
    static propTypes = {
      classes: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
    };

    state = {};

    render() {
      const { classes } = this.props;

      return (
        <Paper className={classes.root}>
          <Typography variant="display1" align="center" gutterBottom>
            Hello!
          </Typography>
        </Paper>
      );
    }
  },
);
