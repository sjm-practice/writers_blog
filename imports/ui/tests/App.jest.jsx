/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */
/* eslint-disable import/no-extraneous-dependencies */
import React from "react";
import renderer from "react-test-renderer";
import App from "../App";

describe("<App />", function() {
  it("matches render snapshot", function() {
    const tree = renderer.create(<App />);
    expect(tree).toMatchSnapshot();
  });
});
