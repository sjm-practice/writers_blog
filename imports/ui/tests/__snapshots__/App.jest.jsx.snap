// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> matches render snapshot 1`] = `
<div
  className="MuiPaper-root-3 MuiPaper-elevation2-7 MuiPaper-rounded-4 App-root-1"
>
  <h1
    className="MuiTypography-root-30 MuiTypography-display1-34 MuiTypography-gutterBottom-48 MuiTypography-alignCenter-44"
  >
    Hello!
  </h1>
</div>
`;
